src/plugins/custom_integrations/server/language_clients/index.ts

```diff
@@ -151,7 +151,8 @@ export function registerLanguageClients(
       description: integration.description,
       type: 'ui_link',
       shipper: 'language_clients',
-      uiInternalPath: integration.docUrlTemplate.replace('{branch}', branch),
+      // Documentation for `main` branches is still published at a `master` URL.
+      uiInternalPath: integration.docUrlTemplate.replace('{branch}', 'master'),
```

Comment:
on second thought, should we use [similar logic](https://github.com/elastic/kibana/blob/e5b279f318eac64660a5f4b157e891339f05ea61/src/core/public/doc_links/doc_links_service.ts#L23) here? because otherwise all versions (7.16, 8.0, etc - when published) will go to `master`

```suggestion
      uiInternalPath: integration.docUrlTemplate.replace('{branch}', branch === 'main' ? 'master' : branch),
```

Reply:
My intention was to just leave this change in the "main" branch and not backport it to other versions. However, I tested and this works, so let's go for it!